custom/auth/ante/fee_burntax.go

```diff
@@ -27,6 +30,37 @@ func (fd FeeDecorator) BurnTaxSplit(ctx sdk.Context, taxes sdk.Coins) (err error
 		taxes = taxes.Sub(distributionDeltaCoins...)
 	}
 
+	if communityTax.IsPositive() {
+
+		// we need to apply a reduced community tax here as the community tax is applied again during distribution
+		// in the distribution module and we don't want to calculate the tax twice
+		// the reduction depends on the oracle split rate as well as on the community tax itself
+		// the formula can be applied even with a zero oracle split rate
+		applyCommunityTax := communityTax.Mul(oracleSplitRate.Quo(communityTax.Mul(oracleSplitRate).Add(sdk.OneDec()).Sub(communityTax)))
+
+		for _, distrCoin := range distributionDeltaCoins {
+			communityTaxAmount := applyCommunityTax.MulInt(distrCoin.Amount).RoundInt()
+			communityTaxCoins = communityTaxCoins.Add(sdk.NewCoin(distrCoin.Denom, communityTaxAmount))
+		}
+
+		distributionDeltaCoins = distributionDeltaCoins.Sub(communityTaxCoins...)
+	}
+
+	if !communityTaxCoins.IsZero() {
+		if err = fd.bankKeeper.SendCoinsFromModuleToModule(
```

Comment:
Can you check if there is a distribution keeper method to be called directly to fund the community pool ?